test/runtests.jl

```diff
@@ -15,6 +15,17 @@ using Test
       for d in 1:5
             @test hcubature(x -> 1, fill(0,d), fill(1,d))[1] ≈ 1 rtol=1e-13
       end
+      @test hcubature_count(x -> 2, (0,0), (2pi, pi))[1] ≈ 4pi^2
```

Comment:
Would be good to check the number of evaluations, which should be the minimal number of the quadrature rule for a constant function: i.e. `count == HCubature.countevals(HCubature.GenzMalik(Val(2)))`

Reply:
Done. I split this test into a new `@testset` with `@test`s for both the value and the count.